.github/workflows/wheels.yml

```diff
@@ -105,10 +110,17 @@ jobs:
             py: cp310
             arch: universal2
 
+          # Windows-latest cp310, we can only include win_amd64 builds as numpy
+          # does not support win
+          # Please see https://cibuildwheel.readthedocs.io/en/stable/options/#archs
+          - os: windows-latest
+            py: cp310
+            arch: AMD64
+
     env:
       CIBW_BUILD: ${{ matrix.py }}-*  # Build every type of image https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
       CIBW_ARCHS: ${{ matrix.arch }}
-      CIBW_TEST_REQUIRES: pytest threadpoolctl numpy
+      CIBW_TEST_REQUIRES: pytest threadpoolctl numpy scipy  # Seems to try install scipy instead
```

Comment:
Sorry, I'm not sure I understand the comment you added. However, I think we actually require scipy by now, so adding this to the requirements make sense.

Reply:
Whoops that can be removed, essentially that was me debugging why it didn't work with `cp10-win32`, turns out it was because numpy didn't provide a wheel for it, so scipy doesn't either. Will remove.

P.s. It's already in the requirements